<launch>
  <param name="input_topic" value="/sense3d/scan/" />
  <param name="cloud_frame_id" value="world" />
  <!--param name="input_topic" value="/stereo_down/points2" />
  <param name="cloud_frame_id" value="stereo_down" /-->

  <param name="grasp_pose_topic" value="/task_priority_pose" />

  <param name="object_pose_topic" value="/object_pose" />

  <arg name="scene" default="specification_scene.xml" />

  <node name="uwsim" pkg="uwsim" type="uwsim" args="--configfile $(find pm_grasp_visualization)/$(arg scene)" output="screen" respawn="false"/>
  <node name="grasp_server_split" pkg="pm_grasp_visualization" type="grasp_server_split" output="screen" respawn="false"/>
  <!--node name="waypoint_server" pkg="pm_grasp_visualization" type="waypoint_server" output="screen" respawn="false"/-->

  <!--  La pose del gripper con la camara se muestra continuamente en /gripper_pose
        La pose del gripper con el mundo se envía al topic /desired_grasp_pose ((geometry_msgs::PoseStamped))
        La pose del gripper con el mundo también se publica en /world to /desired_grasp_pose
        En rviz se puede ver un Marker que representa el cilindro detectado

        Parametros tuneables
          applyVoxelGridFilter la resolución del cloud
          setPlaneSegmentationParams para ver cuánto plano se quita

        En pm_grasp_planning puedes descomentar vispToTF.publish(); para tener más debug de frames
        -->
  <!-- Esta TF es necesaria así que si no está hay que ponerla como se pueda -->
  <node pkg="tf" type="static_transform_publisher" name="fake_broadcaster" args="0 0 1 0 0 0 1 world sense3d 100" />

</launch>
